server/src/main/java/org/opensearch/index/remote/RemoteStorePressureSettings.java

```diff
@@ -187,27 +145,11 @@ public void setMinConsecutiveFailuresLimit(int minConsecutiveFailuresLimit) {
         this.minConsecutiveFailuresLimit = minConsecutiveFailuresLimit;
     }
 
-    public int getUploadBytesMovingAverageWindowSize() {
-        return uploadBytesMovingAverageWindowSize;
-    }
-
-    public void setUploadBytesMovingAverageWindowSize(int uploadBytesMovingAverageWindowSize) {
-        this.uploadBytesMovingAverageWindowSize = uploadBytesMovingAverageWindowSize;
-    }
-
-    public int getUploadBytesPerSecMovingAverageWindowSize() {
-        return uploadBytesPerSecMovingAverageWindowSize;
-    }
-
-    public void setUploadBytesPerSecMovingAverageWindowSize(int uploadBytesPerSecMovingAverageWindowSize) {
-        this.uploadBytesPerSecMovingAverageWindowSize = uploadBytesPerSecMovingAverageWindowSize;
-    }
-
-    public int getUploadTimeMovingAverageWindowSize() {
-        return uploadTimeMovingAverageWindowSize;
+    public int getMovingAverageWindowSize() {
+        return movingAverageWindowSize;
     }
 
-    public void setUploadTimeMovingAverageWindowSize(int uploadTimeMovingAverageWindowSize) {
-        this.uploadTimeMovingAverageWindowSize = uploadTimeMovingAverageWindowSize;
+    public void setMovingAverageWindowSize(int movingAverageWindowSize) {
```

Comment:
the setters seems to have been mistakenly set with public access, pls change this to private access.

Reply:
Also changing the existing getters to be package private